.github/workflows/heartbeat.yaml

```diff
@@ -0,0 +1,22 @@
+name: Commit to keep workflows active
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  heartbeat:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Configure Git
+        run: |
+          git config --global user.email "heartbeat@github.nil"
+          git config --global user.name "Repository Heartbeat"
+
+      - name: Make & push an empty commit
+        run: |
+          git commit --allow-empty -m '[EMPTY] Commit to keep workflows active'
```

Comment:
I don't think we should add this commit on a regular basis without necessity.  
Could you please here check if this repository's HEAD commit is >50 days old and only then do empty commit/push?

Reply:
or may be somewhere in API there is a way to get "last_activity" or alike attribute for the repo to use here instead of just a commit date (may be that timestamp is updated with issues etc activity)

Reply:
@yarikoptic The closest thing I can find in the API is a "`pushed_at`" timestamp field in repository objects.  Also, I'm not actually sure whether issue activity even counts towards keeping workflows active.

Reply:
Let's may be then just use that `pushed_at` for now.

Reply:
or whatever you have already done ;) thanks! Let's proceed

Reply:
we just need to not forget to monitor the repo for a few days so some spurious / not-needed commits  do not pop up